docs/upgrade/services_removal.md

```diff
@@ -0,0 +1,85 @@
+We moved business logic into entities and creation logic into factories.
```

Comment:
I tried to rewrite the sentence

Reply:
:+1: Maybe, it would be nice to explain the word "services" so nobody interchanges that with the Symfony services. I would add "(i.e. *Service classes - not to be interchanged with Symfony services)" after the word "services" :slightly_smiling_face: